admin/extensions.php

```diff
@@ -72,22 +72,25 @@
 								<li>
 									<?php if ( in_array( strtolower( $extension->title->rendered ), array_keys( $installed_plugins ) ) ) : ?>
 
-										<span class="button button-disabled" title="<?php esc_attr_e( 'This extension is already installed', 'backupwordpress' ); ?>"><?php _e( 'Installed', 'backupwordpress' ); ?></span>
+										<span class="button button-disabled" title="<?php esc_attr_e( 'This extension is already installed', 'backupwordpress' ); ?>"><?php esc_html_e( 'Installed', 'backupwordpress' ); ?></span>
 
 									<?php else : ?>
 
-										<a class="install-now button-primary" data-slug="<?php echo esc_attr( $extension->slug ); ?>" href="<?php echo esc_url( $extension->link ); ?>" aria-label="<?php echo esc_attr( sprintf(
-										__( 'Install %s now', 'backupwordpress' ),
+										<a class="install-now button-primary" data-slug="<?php echo esc_attr( $extension->slug ); ?>" href="<?php echo esc_url( $extension->link ); ?>" aria-label="<?php printf(
+										/* translators: Extension name */
+										esc_attr__( 'Install %s now', 'backupwordpress' ),
 										$extension->title->rendered
-										) ); ?>" data-name="<?php echo esc_attr( $extension->title->rendered ); ?>"><?php printf( __( 'Buy Now &dollar;%s', 'backupwordpress' ), $extension->edd_price ); ?></a>
+										); ?>" data-name="<?php echo esc_attr( $extension->title->rendered ); ?>"><?php printf(
+											/* translators: Price */
+											esc_html__( 'Buy Now &dollar;%s', 'backupwordpress' ), $extension->edd_price ); ?></a>
```

Comment:
Alright, so I've raised this track ticket: https://core.trac.wordpress.org/ticket/38179 for HTML5 entities to be added to `$allowedentitynames`.

Meanwhile, I'm using `&#36;` representation for the `$` sign.